GiwazipClient/Views/SettingViewController.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  SettingViewController.swift
+//  GiwazipClient
+//
+//  Created by 김민택 on 2023/02/15.
+//
+
+import UIKit
+
+class SettingViewController: UICollectionViewController {
+
+    // MARK: - Property
+
+    private let settingItems = [["고객 정보 수정"],
+                                ["공지사항", "이용약관", "개발자 정보", "오픈소스 라이브러리", "고객센터", "버전 정보"],
+                                ["시공 마감하기"]]
+
+    private enum Section: CaseIterable {
+        case userSetting
+        case appSetting
+        case roomSetting
+    }
+
+    private lazy var dataSource: UICollectionViewDiffableDataSource<Section, String> = {
+        let cellRegistration = UICollectionView.CellRegistration<UICollectionViewListCell, String>
+        { cell, indexPath, item in
+            var content = cell.defaultContentConfiguration()
+            
+            content.text = item
+
+            if indexPath.section == 1, indexPath.item == 5 {
+                let version = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as! String
+                content.secondaryText = version
+                content.secondaryTextProperties.font = UIFont.systemFont(ofSize: 16)
+                content.secondaryTextProperties.color = .gray
+            }
+
+            content.prefersSideBySideTextAndSecondaryText = true
+
+            if indexPath.section == 2 {
+                content.textProperties.color = .red
+            }
+
+            cell.contentConfiguration = content
+        }
+
+        return UICollectionViewDiffableDataSource<Section, String>(collectionView: collectionView)
+        { (collectionView, indexPath, item) in
+            collectionView.dequeueConfiguredReusableCell(using: cellRegistration,
+                                                         for: indexPath,
+                                                         item: item)
+        }
+    }()
```

Comment:
```suggestion
        return UICollectionViewDiffableDataSource<Section, String>(collectionView: collectionView)
        { (collectionView, indexPath, item) -> UICollectionViewCell? in
            collectionView.dequeueConfiguredReusableCell(using: cellRegistration,
                                                         for: indexPath,
                                                         item: item)
        }
    }()
```

> 아래가 suggest인가요? suggest를 사용하니 뭔가 아래꺼로 수정했다는 느낌이 들어서 확인차 묻습니다.
현재 반영되어 있는 코드는 위 코드로 되어있습니다. 코멘트 확인 후 답변 부탁드립니다.

이 코멘트에 직접 답글을 달 수 없어서 별도의 코멘트로 설명드리자면, 아래가 suggest입니다. 현재는 `-> UICollectionViewListCell?` 부분이 반영되어 있지 않습니다. 해당 부분이 생략된 것이 모호하다고 생각된다면 추가해도 좋을 것 같습니다.